templates/settings.json.j2

```diff
@@ -84,10 +84,14 @@ SPDX-License-Identifier: AGPL-3.0-or-later
     "chatAndUsers": false,
     "lang": "en-gb"
   },
-{% if etherpad_admin_username != "" and etherpad_admin_password != "" %}
+{% if etherpad_admin_username != "" %}
   "users": {
     {{ etherpad_admin_username|to_json }}: {
-      "password": {{ etherpad_admin_password|to_json }},
+      {% if etherpad_admin_password != "" %}
+      "password": "{{ etherpad_admin_password|to_json }}",
+      {% else %}
+      "hash": "{{ etherpad_admin_hash|to_json }}",
+      {% endif %}
```

Comment:
```suggestion
      "password": {{ etherpad_admin_password|to_json }},
      {% else %}
      "hash": {{ etherpad_admin_hash|to_json }},
      {% endif %}
```

`to_json` makes sure to quote the string correctly by itself. Adding extra quotes manually will mean we'll end up with double-quoting.

-----

Not using `to_json` like this could work:

`"password": "{{ etherpad_admin_password }}",`

.. but has the chance of breaking if `etherpad_admin_password` contains a `"`.

That's why we always try to be safe - by using `to_json` and not doing our own quoting manually.

Various roles may still have old configs which do not use `to_json`. I try to fix such occurrences when I see them.